CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerTests.m

```diff
@@ -277,6 +279,40 @@ - (void)testSlotRegistrationRefreshConfiguration {
   OCMVerify([self.configManagerMock refreshConfig:[OCMArg any]]);
 }
 
+#pragma mark - Live Bidding
+
+- (void)testLiveBid_GivenResponseBeforeTimeBudget_ThenBidFromResponseGiven {
+  CR_CdbBid *liveBid = [self givenApiHandlerRespondBid];
+  self.threadManager.isTimeout = NO;
+
+  // Bid Manager returns bid from cdb call
+  [self.bidManager fetchLiveBidForAdUnit:self.adUnit1
+                      bidResponseHandler:^(CR_CdbBid *bid) {
+                        XCTAssertEqualObjects(bid, liveBid);
+                      }];
+  CR_OCMockVerifyCallCdb(self.apiHandlerMock, @[ self.adUnit1 ]);
+
+  // Bid from cdb call has not been cached
+  OCMReject([self.cacheManager setBid:liveBid]);
```

Comment:
liveBid could even be replace by `OCMOCK_ANY` to emphasis that nothing was mocked.

Reply:
Will do.

Reply:
Fixed by a7c8248